packages/widget/src/utils/index.ts

```diff
@@ -46,3 +48,12 @@ export const capitalize = (s: string): string => {
   const rest = s.slice(1);
   return `${firstLetter}${rest}`;
 };
+
+export const validatePolkadotAddress = (address: string): boolean => {
```

Comment:
btw @mpetrunic because we hare using `jsdom` as env for the test, the call to `decodeAddress(address)` function fails because of `Invalid decoded address checksum`. I tested changing the env to node and it passes. Also if you pass `false` to avoid checking checksum it also passes. Here I'm hesitant on how to proceed, do I mock `decodeAddress` just to make the tests pass? Running this on browser context works as expected, since this is the recipe from the `polkadot.js` docs

Reply:
@wainola pushed commit to change happy-dom to jsdom (it seems to not have this issue) but we should work on running in actual browser.